providers/io/osf/normalizer.py

```diff
@@ -0,0 +1,82 @@
+from share.normalize import ctx
+from share.normalize.parsers import Parser
+from share.normalize.links import Delegate, Map, Maybe
+
+
+class Person(Parser):
+    given_name = ctx.embeds.users.data.attributes.given_name
+    family_name = ctx.embeds.users.data.attributes.family_name
+    additional_name = ctx.embeds.users.data.attributes.middle_names
+    suffix = ctx.embeds.users.data.attributes.suffix
+    url = ctx.embeds.users.data.links.html
+
+    class Extra:
+        nodes = ctx.embeds.users.data.relationships.nodes.links.related.href
+        locale = ctx.embeds.users.data.attributes.locale
+        date_registered = ctx.embeds.users.data.attributes.date_registered
+        active = ctx.embeds.users.data.attributes.active
+        timezone = ctx.embeds.users.data.attributes.timezone
+        profile_image = ctx.embeds.users.data.links.profile_image
+
+
+class Contributor(Parser):
+    person = Delegate(Person, ctx)
+    order_cited = ctx('index')
+    cited_name = ctx.embeds.users.data.attributes.full_name
+
+
+class Taxonomy(Parser):
+    name = ctx
+
+
+class Tag(Parser):
+    name = ctx
+    type = Delegate(Taxonomy, ctx)
+
+
+class ThroughTags(Parser):
+    tag = Delegate(Tag, ctx)
+
+
+class Institution(Parser):
+    name = ctx.attributes.name
+    url = ctx.links.self
+
+    class Extra:
+        nodes = ctx.relationships.nodes.links.related.href
+        users = ctx.relationships.users.links.related.href
+        registrations = ctx.relationships.registrations.links.related.href
+        description = ctx.attributes.description
+
+
+class ThroughInstitutions(Parser):
+    institution = Delegate(ctx, Institution)
+
+
+class CreativeWork(Parser):
```

Comment:
Does the api provide a link back to the project?
Do we ever return DOIs or ARK ids?